apps/docs/scripts/components-with-animation.js

```diff
@@ -10,8 +10,8 @@ export const supportedComponents = [
   'Label',
   'Pillar',
   'SectionIntro',
-  'Stack',
   'Statistic',
+  'Stack',
```

Comment:
No clue why this happens from time to time 🤷

Reply:
Yeah, I kind of know why but haven't got around to fixing it yet in the build script 😬... leave it with me, and thanks for the reminder. It's a noop file change FWIW.